docs/guides/: DRY and guide stage normalization (#396)
zbm-dev/zfsbootmenu: docs/guides/_include/define-env.rst

Define disk variables 
---------------------

For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that

> will be configured during the setup. Verify your target disk devices with ``lsblk``.

Reviewer: ```suggestion
will be configured during the setup. Verify your target disk devices with ``lsblk``.

For many users, it is most convenient to place boot files (*i.e.*, ZFSBootMenu and any loader responsible for launching it) on the the same disk that will hold the ZFS pool. However, some users may wish to dedicate an entire disk to the ZFS pool and place these files on a separate device. A USB flash drive provides a convenient mechanism for booting whole-disk pools and is easily recreated if lost. Fortunately, this alternative configuration is easily realized by simply defining a few environment variables differently.
```

Author: I've added a variation of this text to the preamble of this section. Let me know what you think of it.

.. tabs::

   .. group-tab:: Single Disk

      First, define variables that refer to the disk and partition number that will hold boot files:

      .. code-block::

        export BOOT_DISK="/dev/sda"
        export BOOT_PART="1"
        export BOOT_DEVICE="${BOOT_DISK}${BOOT_PART}"

      Next, define variables that refer to the disk and partition number that will hold the ZFS pool:

      .. code-block::

        export POOL_DISK="/dev/sda"
        export POOL_PART="2"
        export POOL_DEVICE="${POOL_DISK}${POOL_PART}"

>    .. group-tab:: Single Disk, external ESP

Reviewer: ```suggestion
   .. group-tab:: Separate Boot Device
```

      First, define variables that refer to the disk and partition number that will hold boot files:

      .. code-block::

        export BOOT_DISK="/dev/sdb"
        export BOOT_PART="1"
        export BOOT_DEVICE="${BOOT_DISK}${BOOT_PART}"

      Next, define variables that refer to the disk and partition number that will hold the ZFS pool:

      .. code-block::

        export POOL_DISK="/dev/sda"
        export POOL_PART="1"

>         export POOL_DEVICE="${POOL_DISK}${POOL_PART}"

Reviewer: To further the DRY goal, you could move the text between the code blocks outside of the tabbed environment. This could be done with a general overview before a single code block that mashes together both var groups, or you could keep the current flow by using two separate groups. As long as changing tabs in one block syncs with with the other, this would provide a decent experience.

Define disk variables 
---------------------

For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that
will be configured during the setup.

For many users, it is most convenient to place boot files (*i.e.*, ZFSBootMenu and any loader responsible for
launching it) on the the same disk that will hold the ZFS pool. However, some users may wish to dedicate an entire

> disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient mechanism for storing the

Reviewer: ```suggestion
disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient home for the
```

Define disk variables 
---------------------

For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that
will be configured during the setup.

For many users, it is most convenient to place boot files (*i.e.*, ZFSBootMenu and any loader responsible for
launching it) on the the same disk that will hold the ZFS pool. However, some users may wish to dedicate an entire

> disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient mechanism for storing the
> EFI boot partition (**ESP**). Fortunately, this alternative configuration is easily realized by simply defining a

Reviewer: This section looks good, but the EFI referenes should be dropped because this block also appears in the syslinux/MBR guide.

```suggestion
disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient mechanism for storing the
boot partition. Fortunately, this alternative configuration is easily realized by simply defining a
```
